src/engine/source/store/interface/store/utils.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef _STORE_UTILS_H
+#define _STORE_UTILS_H
+
+#include <memory>
+#include <optional>
+#include <variant>
+
+#include "store/istore.hpp"
+#include <error.hpp>
+#include <json/json.hpp>
+#include <name.hpp>
+
+/**
+ * @brief Store functionallity.
+ *
+ */
+namespace store
```

Comment:
Is a different approach possible for these functions? I find it confusing to have:
`store->get(xx)` and `store::get(store, xx)`.